### PR TITLE
Toros pool card

### PR DESCRIPTION
**Pull request by Rickk137**

### CODE REVIEW

**Review by noisekit — APPROVED** [review]
Looking good. Let's get @thisispixelperfect approval

**Comment by thisispixelperfect** [comment]
we're missing the icon for yield generating asset 
Screenshot 2024-10-03 at 13 06 25

**Review by noisekit — APPROVED** [review]

File: liquidity/ui/src/components/Pools/PoolCards/Specifics.tsx [hunk 1]
```diff
@@ -1,44 +1,36 @@
...
-            <path
-              d="M10.7261 7.08202L11.649 5.89844L12.7141 6.94727L14.0214 6.2066L14.6009 7.58549L16.0896 7.39984L16.0975 8.89921L17.5421 9.29568L16.9747 10.687L18.1571 11.6064L17.1029 12.6769L17.8408 13.9793L16.4594 14.5679L16.6424 16.0538L15.1423 16.0704L14.7408 17.5148L13.3537 16.9543L12.4308 18.138L11.3657 17.0891L10.0615 17.8289L9.47887 16.4509L7.99325 16.6357L7.98548 15.1363L6.54081 14.7398L7.10506 13.3493L5.92578 12.4292L6.97999 11.3586L6.24205 10.0562L7.62041 9.46846L7.44058 7.98177L8.93746 7.96601L9.33895 6.52157L10.7261 7.08202Z"
-              fill="#69D7FF"
-            />
-          </svg>
-
+      <Flex alignItems="center" gap={2}>
+        {collateral?.toUpperCase() === 'wstETH'.toUpperCase() && (
+          <Tooltip textAlign="left" label="Yield on yield" hasArrow>
+            <svg
```
Comment (noisekit) [inline liquidity/ui/src/components/Pools/PoolCards/Specifics.tsx:17]: Extract svgs into "*.svg" files and import. We have support for svg files.
Can be separate PR not this one.

File: liquidity/ui/src/components/Pools/PoolCards/PoolRow.tsx [hunk 2]
```diff
@@ -223,7 +223,7 @@ export const PoolRow = ({ pool, network, apr, collateralType, collateralPrices }
             </Text>
           </Flex>
           <Flex alignItems="center" justifyContent="flex-end" width="121px" textAlign="right">
-            <Specifics network={network} collateralType={collateralType} />
+            <Specifics network={network} collateral={collateralType.symbol} />
```
Comment (noisekit) [inline liquidity/ui/src/components/Pools/PoolCards/PoolRow.tsx:226]: We'd better keep collateralType object as all the comparisons will be made with addresses not symbols (soon...)

**Comment by noisekit** [comment]
> we're missing the icon for yield generating asset

Fixed now

File: liquidity/ui/src/components/Pools/PoolCards/PoolRow.tsx (hunk 2 above)
Comment (noisekit) [reply liquidity/ui/src/components/Pools/PoolCards/PoolRow.tsx]: fixed

File: liquidity/ui/src/components/Pools/PoolCards/Specifics.tsx (hunk 1 above)
Comment (noisekit) [reply liquidity/ui/src/components/Pools/PoolCards/Specifics.tsx]: fixed
